site/cypress/e2e/pronouns_change.spec.js

```diff
@@ -0,0 +1,76 @@
+
+describe('Tests cases abut changing user pronouns', () => {
+
+    let oldPronouns = '';
+
+    it('Tests changing pronouns for student', () => {
+        cy.visit('/user_profile');
+        cy.login('student');
+
+        //open pronouns form
+        cy.get('#pronouns_val').click();
+        const e = cy.get('#user-pronouns-change');
+
+        //save old pronouns
+        e.then(($pronounsInput) => {
```

Comment:
```suggestion
        cy.get('@e').then(($pronounsInput) => {
```

Reply:
Is cy.get('@e') preferrable in any way? otherwise it may make the code less readable

Reply:
[Assigning Return Values](https://docs.cypress.io/guides/references/best-practices#Assigning-Return-Values)

It is just the best practice for cypress. In theory (and best practice) const, let, and var shouldn't really ever be used.

Reply:
I feel like it's bad since it's hard to read for someone unfamiliar with cypress, but since it's declared as the way to do it I will